airflow/bin/cli.py

```diff
@@ -72,15 +72,30 @@ def sigquit_handler(sig, frame):
     print("\n".join(code))
 
 
-def setup_logging(filename):
-    root = logging.getLogger()
+def setup_file_logging(filename, fmt=settings.SIMPLE_LOG_FORMAT):
     handler = logging.FileHandler(filename)
```

Comment:
Since github won't let us update somebody else's PR to point to a different branch, my colleague, Luke, has created a new PR for this here: https://github.com/apache/incubator-airflow/pull/1921/files

Could someone with permission please close this PR as it is superseded by that one?